README.md

```diff
@@ -1,81 +1,17 @@
-        ],
-    ],
-```
-
-Extensions that dropped support for TYPO3 v8 (or keeps separate branches) and did not migrate
-to doctrine in its v9 version, should list typo3db_legacy in the 'depends' section of
-ext_emconf.php:
-
-```
-    'constraints' => [
-        ...
-        'depends' => [
-            'typo3db_legacy' => '1.0.0-1.1.99',
-        ],
-    ],
-```
-
-## Configuration
-The extension consumes the same 'Default' configuration from TYPO3_CONF_VARS as the
-doctrine based API. The connection is initialized in the extension's ext_localconf.php file.
-
-## Current state
-The latest version here reflects a feature-complete state. There are bugs, we know,
-there are possible feature requests - we know. But it's highly likely that this
-extension gets no new features, unless somebody steps up and continues the development
-(see further below).
-
-## Contribution
-Feel free to submit any pull request, or add documentation, tests, as you please.
-We will publish a new version every once in a while, depending on the amount of changes
-and pull requests submitted.
-
-If you want to keep adding features, and keep typo3db_legacy compatible with the latest
-TYPO3 versions, feel free to contact Benni Mack.
-
-## License
-The extension is published under GPL v2+, all included third-party libraries are
-published under their respective licenses.
-
-## Authors
-A lot of contributors have been working on this area while this functionality was part of
-the TYPO3 Core. This package is now maintained by a loose group of TYPO3 enthusiasts inside
-the TYPO3 Community. Feel free to contact Benni Mack (benni.mack@typo3.org) for any questions
-regarding "typo3db_legacy".
+[![Latest Stable Version](https://poser.pugx.org/friendsoftypo3/typo3db-legacy/v/stable.svg)](https://extensions.typo3.org/extension/typo3db_legacy/)
+[![TYPO3](https://img.shields.io/badge/TYPO3-11-orange.svg?style=flat-square)](https://get.typo3.org/version/11)
+[![TYPO3](https://img.shields.io/badge/TYPO3-10-orange.svg?style=flat-square)](https://get.typo3.org/version/10)
+[![TYPO3](https://img.shields.io/badge/TYPO3-9-orange.svg?style=flat-square)](https://get.typo3.org/version/9)
+[![Total Downloads](https://poser.pugx.org/friendsoftypo3/typo3db-legacy/d/total.svg)](https://packagist.org/packages/friendsoftypo3/typo3db-legacy)
+[![Monthly Downloads](https://poser.pugx.org/friendsoftypo3/typo3db-legacy/d/monthly)](https://packagist.org/packages/friendsoftypo3/typo3db-legacy)
+
+# TYPO3 extension `typo3db_legacy`
+
+This extension provides `$GLOBALS['TYPO3_DB']` as backward compatibility with
+legacy functionality of extensions that are not yet fully migrated to doctrine.
+
+|                  | URL                                                                |
+|------------------|--------------------------------------------------------------------|
+| **Repository:**  | https://github.com/FriendsOfTYPO3/typo3db_legacy                   |
```

Comment:
should we remove the ":" looks better without them in the table (my personal opinion)

Reply:
we are at the very end of this standardization run - we should move all further discussions on this standard to a next issue and run: this is the [current standard](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#about-file) and you could create a new standardization issue at the [T3DocTeam](https://github.com/TYPO3-Documentation/T3DocTeam/issues) repository, if you like.